.changeset/six-islands-exercise.md

```diff
@@ -0,0 +1,17 @@
+---
+"@navikt/ds-react": minor
+"@navikt/ds-css": minor
+---
+
+ErrorSummary: :sparkles: new prop to prefix error with a warning icon.
+Typography: replace warning dot with warning icon
+DateInput: replace warning dot with warning icon
+Combobox: replace warning dot with warning icon
+Combobox: replace warning dot with warning icon
+ConfirmationPanel: replace warning dot with warning icon
+Fieldset: replace warning dot with warning icon
+DropZone: replace warning dot with warning icon
+Search: replace warning dot with warning icon
+Select: replace warning dot with warning icon
+Textarea: replace warning dot with warning icon
+TextField: replace warning dot with warning icon
```

Comment:
I think this will bloat the changelog. Can we get this down to one or two lines? 🤔 (Also, what actually happens when you have multiple lines in one changeset? I guess we can try and see.)

Suggestions:

> ErrorMessage: :sparkles: new prop to prefix error with a warning icon.
> DateInput/Combobox/ConfirmationPanel/CheckboxGroup/RadioGroup/FileUpload.Dropzone/Search/Select/Textarea/TextField: replace dot with warning icon in error message.

> ErrorMessage: :sparkles: new prop to prefix error with a warning icon.
> Replace dot with warning icon in error message in all form components.

Alternatively "all **relevant** (form) components" if there are form components that does not use ErrorMessage, or some of the affected components are not considered form components.

Reply:
`form_components(*): message` 🤔 (like a function call)